### DISCUSSION TITLE
How to change monitor resolution for borderless fullscreen?

### DISCUSSION THREAD

**Discussion by aedm** [rust-windowing/winit#2808] [Q&A] [answered]
I'd like to run an app in borderless full screen. But I also want to specify the screen resolution, too. 
Is there a way to change the screen resolution for borderless mode using Winit? 
(Or any other platform-independent way in Rust? What's my best option?)

I'm currently using exclusive mode, but it has some compatibility issues. Specifically, screen recording using Nvidia's tools doesn't quite work, and screen mirroring has some glitches when GSync is enabled.

**Comment by kchibisov** [comment, marked as answer]
I don't think so, I know how to do that on Wayland for example and know that it's possible, but the platform doesn't have the exclusive fullscreen.

Be aware that exclusive fullscreen might be not something you really want, what type of software are you developing? If it's a game exclusive fullscreen is something you'd want, but for everything else you better scale everything yourself.

If the goal is to let me render at 1920x1080, but let compositor scale to 2560x1440, I think only Wayland has such concept.
